modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/info/item/provider/test/ObjectEntryInfoItemFieldValuesProviderTest.java

```diff
@@ -241,9 +239,64 @@ public void testObjectEntryInfoItemFieldValuesProvider() throws Exception {
 			).build(),
 			false);
 
-		InfoItemFieldValues infoItemFieldValues =
+		// Getting infoItemFieldValues passing the HttpServletRequest without a
+		// ThemeDisplay attribute
+
+		_serviceContext.setRequest(_mockHttpServletRequest);
+
+		ServiceContextThreadLocal.pushServiceContext(_serviceContext);
+
+		InfoItemFieldValues infoItemFieldValues1 =
 			infoItemFieldValuesProvider.getInfoItemFieldValues(objectEntry);
 
+		_assertValues(
+			fileEntry, infoItemFieldValues1, objectAction,
+			parentTextObjectFieldNameValue);
+
+		// Getting infoItemFieldValues passing the HttpServletRequest with a
+		// ThemeDisplay attribute
+
+		_serviceContext.setRequest(_getHttpServletRequest(_group));
+
+		ServiceContextThreadLocal.pushServiceContext(_serviceContext);
```

Comment:
@vngoliveira notice that you are calling the "push" method twice here. So you also need to call the "pop" twice.